client/webserver/site/src/html/settings.tmpl

```diff
@@ -17,6 +17,13 @@
         Show pop-up notifications
       </label>
     </div>
+    <div id="dexBox">
+        <div class="bg2 px-2 py-1" id="dexInfo">
+            Dex Address: <span class="dexInfoHost"></span><br>
```

Comment:
You should just iterate through the `.UserInfo.Exchanges` and add the elements here.